tests/phpunit/testcases/core/themeCompatibility.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @group core
+ * @group core-theme-compatibility
+ */
+class BP_Tests_Theme_Compatibility_Functions extends BP_UnitTestCase {
+
+	/**
+	 * @group bp_current_theme_supports
```

Comment:
I'd move this to the top so that any test in this file is grouped. Better than adding each time for each, new or existing, test.